docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/gcp.md

```diff
@@ -96,13 +96,13 @@ For more information, see [Creating and managing service accounts](https://cloud
 
 ## Create a firewall rule
 
-In order to access Yugabyte Platform from outside the GCP environment, you have to enable firewall rules. At a minimum, you need the following:
+In order to access Yugabyte Platform from outside the GCP environment, you have to enable firewall rules. At a minimum, you need to be able to do the following:
 
-- Access the Yugabyte Platform instance over SSH (port `tcp:22`)
-- Check, manage, and upgrade Yugabyte Platform (port `tcp:8800`)
-- View the Yugabyte Platform UI (port `tcp:80`)
+- Access the Yugabyte Platform instance over SSH (port `tcp:22`).
```

Comment:
On AWS page you didn't add periods to this list

Reply:
Added periods